app/controllers/admin/editions_controller.rb

```diff
@@ -205,8 +202,8 @@ def get_layout
     end
   end
 
-  def fetch_version_and_remark_trails(next_release: true)
-    if preview_design_system?(next_release:)
+  def fetch_version_and_remark_trails(govuk_design_system: false)
+    if govuk_design_system
```

Comment:
**No need to change this**, but it's just occurred to me that we probably don't need any parameters on this method since we could just call `get_layout` to work that out (making that the single source of truth).

i.e.

```suggestion
  def fetch_version_and_remark_trails
    if get_layout == "design_system"
```

Reply:
great shout. Will update that now